httprpc-server/src/main/resources/org/httprpc/api.html

```diff
@@ -148,6 +151,25 @@
       color: var(--color-neutral);
     }
   </style>
+  <script>
+    document.addEventListener('DOMContentLoaded', function(event) { 
+      const tocLinks = document.querySelectorAll('#toc a');
+      const highlightActive = function(){
```

Comment:
This looks good - only suggestions are minor:
* Use consistent spacing; e.g. spaces before "{" and "("
* Don't use abbreviations in variable names; e.g. change `curHash` to just `hash`